Dockerfile

```diff
@@ -19,6 +19,12 @@ RUN chmod +x /actions-runner/install_actions.sh \
 
 COPY token.sh entrypoint.sh /
 RUN chmod +x /token.sh /entrypoint.sh
-
 ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
even though i answered outside the thread and order doesn't matter it's still best practice to keep ENTRYPOINT at the end especially after RUN and USER

Reply:
Sure, I've moved the `ENTRYPOINT` after the user creation.  I've also combined the two `RUN`s above to prevent Sider from complaining, but let me know if you'd rather keep them separate.